src/core/core.tooltip.js

```diff
@@ -604,11 +608,18 @@ module.exports = function(Chart) {
 			}
 
 			ctx.fillStyle = mergeOpacity(vm.backgroundColor, opacity);
+			ctx.lineWidth = vm.borderWidth;
+			ctx.strokeStyle = vm.borderColor;
+
 			ctx.beginPath();
 			ctx.moveTo(x1, y1);
 			ctx.lineTo(x2, y2);
 			ctx.lineTo(x3, y3);
+			ctx.stroke();
+			ctx.fill();
 			ctx.closePath();
+
+			helpers.drawRoundedRectangle(ctx, ptX, ptY, size.width, size.height, vm.cornerRadius);
```

Comment:
@marcelotedeschi I like the idea of drawing together as one path and making it into a single path.

I also like the idea of calling `drawCaret` inside of `drawBackground`. I'm happy to look at PRs of either solution :smile: